client/extensions/woocommerce/app/settings/payments/payment-method-edit-form-toggle.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from 'components/forms/form-toggle';
+
+const PaymentMethodEditFormToggle = ( { checked, name, onChange } ) => {
+	const onChangeHandler = () => {
+		const fakeEvent = {
+			target: {
+				name,
+				value: checked ? 'no' : 'yes'
+			}
+		};
+		onChange( fakeEvent );
+	};
+
+	return (
+		<FormToggle
```

Comment:
We should pass down any other passed props so that this FormToggle can support all other FormToggle props. 

Here is an example of what I mean: https://github.com/Automattic/wp-calypso/blob/c19d6fa3b512621130be715cf603094e519a9086/client/extensions/woocommerce/components/form-click-to-edit-input/index.js#L74